CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ include(GNUInstallDirs)
 # This helps to build vendor projects with or without any patching. Also if any
 # files are changed in vendor projects those can be retained with this option.
 option(ANDROID_TOOLS_PATCH_VENDOR "Patch vendor projects using patches directory" ON)
+option(ANDROID_TOOLS_USE_SYSTEM_FMT "Use system provided fmt library instead of bundled one" OFF)
```

Comment:
android-tool should target to run with system libraries as much as possible. And better if this flag default is `use system libraries`. this way it promotes simplest/lightest solution, and only if it does not work then use the bundled solution.

It also makes sense to swap the flag and call it `ANDROID_TOOLS_USE_BUNDLED_FMT` so the default option value is `OFF`.

Reply:
Would not that break existing default behavior? Also, not all Linux distributions have save fmt version https://repology.org/project/fmt/versions. For example, android-tools 35.0.x uses fmt version 10.2.0.

Reply:
The version skew is fine and it happens all the time. But if the system version does not work then the package manager will need to set the flag you are introducing.

Reply:
The `ANDROID_TOOLS_USE_BUNDLED_FMT` option has to be disabled in CI due to that version skew. Otherwise, it may break in future.

Reply:
Done.